### PR TITLE
[CP-2425][Multidevice] Implement Handling of Code Review Comments and Decision-Making for Task CP-2162

### PR DESCRIPTION
**Pull request by dkarski**
JIRA Reference: [CP-2425]

### :memo: Description ️

- cac7990e - Removed doubled check via not needed `alreadyInitializedDevices`
- e0c60791 - Removed IPC dependency injection from device-manager based on https://appnroll.atlassian.net/wiki/spaces/CP/pages/2186936572/ADR+Utilizing+Direct+Import+of+Dependencies

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2425]: https://appnroll.atlassian.net/browse/CP-2425?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mkurczewski — APPROVED** [review]

File: libs/shared/utils/src/lib/answer-main.ts [hunk 1]
```diff
@@ -4,11 +4,10 @@
  */
 
 import { ipcRenderer } from "electron-better-ipc"
-import { CallRendererEvent } from "device/adapters"
 
-export const answerMain = (
-  event: CallRendererEvent,
-  callback: (data: unknown) => void | PromiseLike<void>
+export const answerMain = <DataType = unknown>(
+  event: string,
```
Comment (lkowalczyk87) [inline libs/shared/utils/src/lib/answer-main.ts:9]: why did you remove CallRendererEvent type?

Comment (dkarski) [reply libs/shared/utils/src/lib/answer-main.ts:9]: I did it this way because I wanted to provide more flexibility in defining events. Using a string instead of CallRendererEvent allows for easier adaptation and greater universality of the function. I thought this would make things easier for various applications, without limiting ourselves to pre-defined event types. It was about making things simpler and more flexible.

Comment (dkarski) [reply libs/shared/utils/src/lib/answer-main.ts]: https://github.com/mudita/mudita-center/pull/1765/commits/cad0360218a266aaa30ba5a1a8386855c6b65a43

**Review by lkowalczyk87 — APPROVED** [review]
